next.config.js

```diff
@@ -7,6 +7,7 @@ const nextConfig = {
       "avatars.githubusercontent.com",
       "user-images.githubusercontent.com",
       "cdn.nhcarrigan.com",
+      "d1aettbyeyfilo.cloudfront.net"
```

Comment:
Please revert the changes in this file. Let us know if you need any help!


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix